model-governance/model_governance_graph.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Not sure how this <code>source</code> and <code>iamRoleArn</code>  and <code>region</code>    is passing CI.... </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>